code/modules/mob/living/living_defines.dm

```diff
@@ -58,3 +58,6 @@
 	var/unique_name = 0 //if a mob's name should be appended with an id when created e.g. Mob (666)
 
 	var/list/butcher_results = null
+
+	var/stun_absorption = FALSE //If all incoming stuns are being absorbed

```

Comment:
I implemented the same power without snowflake code with events: https://github.com/d3athrow/vgstation13/wiki/Events:-writing-less-shit-code